### PR TITLE
Get dqm code ready for cron job

### PR DESCRIPTION
**Pull request by sweng66**

### CODE REVIEW

**Review by valearna — APPROVED** [review]
Everything looks good to me, thanks @sweng66

File: agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py [hunk 1]
```diff
@@ -66,6 +66,9 @@ def read_data_and_load_references(db_session, json_data, journal_to_resource_id,
     for entry in json_data:
 
         primaryId = set_primaryId(entry)
+        crossRef = db_session.query(CrossReferenceModel).filter_by(curie=primaryId).one_or_none()
```
Comment (azurebrd) [inline agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py:69]: I'm not sure what this is for, seems to be avoid entering something if it's already there ?  If so, should it be only if is_obsolete==false ?

**Review by azurebrd — COMMENTED** [review]
Thanks @sweng66 and @valearna !  I just had that one comment about the xref filter for is_obsolete

File: agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py (hunk 1 above)
Comment (sweng66) [reply agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py:69]: Good catch! @azurebrd

Comment (sweng66) [reply agr_literature_service/lit_processing/data_ingest/post_reference_to_db.py]: Fixed =) @azurebrd

**Review by azurebrd — APPROVED** [review]
Thanks @sweng66 !

**Pull request merged**